Sources/SpeziScheduler/Scheduler.swift

```diff
@@ -170,37 +157,12 @@ public class Scheduler<Context: Codable>: NSObject, UNUserNotificationCenterDele
     
     
     // MARK: - Helper Methods
-    private func persistChanges() async {
-        await localStorageLock.enter {
-            do {
-                try self.localStorage.store(self.tasks, storageKey: Constants.taskStorageKey)
-            } catch {
-                os_log(.error, "Spezi.Scheduler: Could not persist the tasks of the scheduler module: \(error)")
-            }
-        }
-    }
-    
-    private func sendObjectWillChange(skipInternalUpdates: Bool = false) async {
-        os_log(.debug, "Spezi.Scheduler: Object will change (skipInternalUpdates: \(skipInternalUpdates))")
-        if skipInternalUpdates {
-            await MainActor.run {
-                self.objectWillChange.send()
-            }
-        } else {
-            self.updateTasks()
-            await updateScheduleNotifications()
-            await persistChanges()
-            await MainActor.run {
-                self.objectWillChange.send()
-            }
-        }
-    }
-    
-    
     @objc
     private func timeZoneChanged() {
-        _Concurrency.Task {
-            await sendObjectWillChange()
+        _Concurrency.Task { @MainActor in
+            logger.debug("TimeZone changed!")
+            self.scheduleTasks()
+            await updateScheduleNotifications()
         }
     }
```

Comment:
I think we can remove this here as we do no longer track this?

Reply:
You are right. Totally forgot this one 👍